README.md

```diff
@@ -2,100 +2,100 @@
-## 🛠️ How to install
-
-**🎉We now support installing via Vue CLI 3.0 🎉**
-
-### Install with Vue-CLI
-
-1. After create your application with Vue-CLI, navigate to the created app folder, and install Cloudinary SDK by:
-
-```bash
-vue add cloudinary
-```
-
-2. Set up your cloudName and pick the components to use (or use all 😃)
-
-![Set up with cloudName and options](http://bit.ly/2WSKTf0)
-
-A `cloudinary.js` file will be added to your src directory, same level with your `main.js` file. Feel free to customize it if you like. And have fun with Cloudinary! 🤟
-
-More information on the plugin, check out [our Vue-CLI plugin for Cloudinary Repo](https://github.com/cloudinary/vue-cli-plugin-cloudinary)
-
-## Manual install with `npm` or `yarn`
-
-Install the package in your project with
-
-```bash
-npm install cloudinary-vue
-
-#OR
-
-yarn add cloudinary-vue
-```
-
-## 💻 Configure
-
-### Configure Cloudinary options
-
-1. In your `main.js` file (or the main entrance file of your application)
-
-```jsx
-import Vue from "vue";
-import Cloudinary from "cloudinary-vue";
-```
-
-2. You can setup Vue to use Cloudinary plugin without any configuration as below
-
-```jsx
-Vue.use(Cloudinary);
-```
-
-**Or** setup with some global Cloundinary configurations, such as `cloudName`
-
-```jsx
-Vue.use(Cloudinary, {
-  configuration: { cloudName: "demo" }
-  //             ^ cloudinary configuration options
-});
-```
-
-3. Globally you can also select and register which Cloudinary components you'd like to use in your app in an **array of components**:
-
-```jsx
-import Cloudinary, { CldImage } from "cloudinary-vue";
-
-// specify which components to install in an array
-Vue.use(Cloudinary, {
-  configuration: { cloudName: "demo" },
-  components: [ CldImage ]
-});
-```
-
-**Or** define them in **object of components**, and rename any component if needed.
-
-```jsx
-import Cloudinary, { CldImage, CldTransformation } from "cloudinary-vue";
-
-Vue.use(Cloudinary, {
-  configuration: { cloudName: "demo" },
-  components: {
-    CldImage,
-    CldXf: CldTransformation
-    //^ a custom name
-  }
-});
-```
-
-**Notes**: By default, if there is no `components` defined, the plugin will _automatically_ install all available Cloudinary components.
-
-### General usage
+Cloudinary is a [cloud-based service](https://cloudinary.com/solutions) that provides an end-to-end image and video management solution
+including uploads,storage, manipulations, optimizations and delivery. 
+All your media resources are optimized and delivered through a fast CDN using industry best practices.
+
+Using Cloudinary's Vue.js SDK, you can easily perform smart image and video manipulations 
+using code that integrates seamlessly with your existing Vue.js application, and deliver optimized and responsive media to your users.
+
+This Readme provides basic installation and usage information. For the complete documentation, see the [Vue.js SDK guide](https://cloudinary.com/documentation/vue_integration)_
```

Comment:
Something in the layout changed here. why?

Reply:
this new 'layout' forces line breaks, the end result looks exactly the same but it's written in a way that makes sense to edit.